### ISSUE TITLE
Does not remove base file.

### ISSUE THREAD

**Issue by jmdiazlr** [feincms/django-imagefield#6]
It removes all generated files but not the base file when we clear the corresponding imagefield in the form and hit save.

**Comment by matthiask** [comment]
Yes, that's true. Django's file fields generally do not remove the base file; I think it would be unexpected and a possible source of data loss if we started removing base files ourselves.
